packages/foam-vscode/__mocks__/vscode.ts

```diff
@@ -11,6 +12,9 @@ https://github.com/microsoft/vscode-test/issues/37#issuecomment-584744386
 
 const vscode = {
     // Add values and methods as needed for tests
+
+    // Keep env the same, no need to mock this.
+    env: env
```

Comment:
can you add an explanation for this?

Reply:
This has been removed, seems it wasn't necessary after all!